app-emulation/docker/files/dockerd

```diff
@@ -64,6 +68,9 @@ maybe_disable_selinux() {
             # Leave enabled for everything else.
             ;;
     esac
+    if [[ -n "${ARG_SELINUX}" ]]; then
```

Comment:
Let's drop this hunk and instead make USE_SELINUX="--selinux-enabled" and maybe_disable_selinux conditional on -z "${ARG_SELINUX}" similar to how select_docker_driver is happening.

Thanks!